pages/planning-policies/org-policies/index.md

```diff
@@ -267,7 +267,7 @@ We will review this policy <mark>[review period]</mark> on or before the <mark>[
 
 ## Next steps: Maintaining your policy {#maintain}
 
-Keep your policy accurate and up to date. Regularly review your policy if it includes milestones.  This is also true if it references particular standards by version number. Reviews should check that milestones have been met. It should also check if changes are required due to new or updated standards.
+Keep your policy accurate and up to date. Regularly review your policy to check that it includes milestones.  This is also true if it references particular standards by version number. Reviews should check that milestones have been met. It should also check if changes are required due to new or updated standards.
```

Comment:
This isn't correct. A policy is not being reviewed to add in milestones. You would review any milestones that you had included in your policy. Ideally there should be some milestones for improvement but possibly not.

Reply:
O, I really misunderstood this sentence then :sweat_smile: I've reverted this change now. Thanks!